src/rules/no_this.coffee

```diff
@@ -0,0 +1,16 @@
+
+module.exports = class NoThis
+    rule:
+        name: 'no_this'
+        description: '''
+        This rule prohibits 'this'.
+        Use '@' instead.
+        '''
+        level: 'ignore'
+        message: 'Don\'t use \'this\''
```

Comment:
I recommend using double quotes so you don't have to escape the single quotes: `"Don't use 'this'"`

This is actually a special case of [`no_unnecessar_double_quotes`](https://github.com/clutchski/coffeelint/blob/master/src/rules/no_unnecessary_double_quotes.coffee#L45)